### ISSUE TITLE
multi-qr-code-reader support

### ISSUE THREAD

**Issue by xujiandream** [zxing-js/library#571]
Hi, repo owner, I found there is a feature branch feature/multi-qr-code-reader support scanning multi qr-code from one picture. Can you  merge this to release branch, I do need this feature on my project. Thanks a lot!
CC: @werthdavid

**Comment by werthdavid** [comment]
I will look through that PR when I find time

**Comment by L-Rocket** [comment]
Is this feature available in the latest package?

**Comment by L-Rocket** [comment]
hello, I wonder if this feature is available in the latest package?

**Comment by werthdavid** [comment]
No, sorry. this needs quite some time which I couldn't find yet...

**Comment by L-Rocket** [comment]
For someone who needs this, I have made one. Welcome to give it a try! 

https://l-rocket.github.io/

**Comment by werthdavid** [comment]
Nice. @L-Rocket what is that based on? what's the license of the project? It's made in vue.js right?

**Comment by L-Rocket** [comment]
Hi! 
1. The project is built upon the 'qr-scanner-wechat' package, which is capable of detecting one of the clearest QR code. We first scan the image to get the clearest QR code. We then cover the detected area to search for the second clearest QR code, and continue this process until no QR code is found in the image.
2. I plan to open-source it whenever I am available.
3. Absolutely! I utilized Vue.js to develop the web interface.
4. Additionally, I intend to use the methods I mentioned above to rewrite another version using this library. This means it will be capable of detecting other types of codes, such as barcodes, which 'qr-scanner-wechat' cannot handle.

**Comment by L-Rocket** [comment]
I've made it open source ! 
Welcome to have a look!
https://github.com/L-Rocket/Multi-qr-code-reader
